host/sgx/calls.c

```diff
@@ -421,9 +420,10 @@ static oe_result_t _handle_ocall(
 
     oe_log(
         OE_LOG_LEVEL_VERBOSE,
-        "%s 0x%x OE_OCALL: %s\n",
+        "%s 0x%x %s: %s\n",
         enclave->path,
         enclave->addr,
+        func == OE_OCALL_CALL_HOST_FUNCTION ? "EDL_OCALL" : "OE_OCALL",
```

Comment:
+1. And in a future PR we would extend `oe_ocall_str` to return the actual EDL function name instead of "OE_OCALL_CALL_HOST_FUNCTION".

Reply:
I had a look at this, there is no easy way to do it now, the names just aren't captured. We'd need to change the way generation works etc, this is definitely for another PR.